### ISSUE TITLE
Typescript DTO generation fails to compile if metadata string span multiple lines, or if two classes in different namespaces have the same names

### ISSUE THREAD

**Issue by bumblebeeman** [ServiceStack/Issues#364]
The title pretty much sums it up.

We can't use the ServiceStack TS generation tool to generate a DTO based on our relatively complex API for Typescript, because we have a number of metadata descriptions which are multi-line strings, which the TS conversion takes literally.

Also,  in some instances we have classes in different namespaces with the same names, which the TS compiler considers as duplicate.

I have had a brief look at the code for the tool which generates the TS, and would be willing to spend a bit of time debugging, seeing if I could tidy things up, if you would be willing to consider a pull request?

I can of course generate one manually, using the code generated by the tool as a guide, but it would be nice to be able to generate directly.

**Comment by mythz** [comment]
The unique DTO Names is a [limitation of Add ServiceStack Reference](https://github.com/ServiceStack/ServiceStack/wiki/Add-ServiceStack-Reference#limitations) feature. DTO's need to be unique for the generated DTO's to be used in different languages.

Are you using the latest v4.0.46 release? cause the multiline string in attributes should be resolved, if you are can you include an example of a Request DTO with the problem?

**Comment by bumblebeeman** [comment]
Ok so just tested against .46, [still seeing the multi line issue.](http://imgur.com/VX6TKSe).

I understand about the other issue, regarding unique DTO's, will keep that in mind.

**Comment by mythz** [comment]
ok apologies [the fix was added just after](https://github.com/ServiceStack/ServiceStack/commit/16179231049928032824dcc56a1dcc40c8c205aa) the v4.0.46 release.

This should be fixed with the [v4.0.47 pre-release packages on MyGet](https://github.com/ServiceStack/ServiceStack/wiki/MyGet).

I've just tested it with this Request DTO:

```csharp
[Api(@"Multi 
Line
Class")]
public class HelloMultiline
{
    [ApiMember(Description = @"Multi 
Line 
Property")]
    [DataMember]
    public string Overflow { get; set; }
}
```

Which produces:

```typescript
/**
* Multi LineClass
*/
// @Api("Multi LineClass")
interface HelloMultiline
{
    // @DataMember
    // @ApiMember(Description="Multi Line Property")
    overflow?: string;
}
```

let me know if v4.0.47 is still an issue.

**Comment by bumblebeeman** [comment]
Fantastic, thankyou!
